src/microbe_stage/MicrobeColony.cs

```diff
@@ -45,6 +52,34 @@ public Microbe.MicrobeState State
     [JsonProperty]
     public Microbe Master { get; private set; }
 
+    /// <summary>
+    ///   The total hex count from all members of this colony.
+    /// </summary>
+    [JsonIgnore]
+    public float HexCount
+    {
+        get
+        {
+            if (hexCountDirty)
+                UpdateHexCount();
+            return hexCount;
+        }
+    }
+
+    /// <summary>
+    ///   The accumulation of all the colony member's <see cref="Microbe.EngulfSize"/>.
+    /// </summary>
+    [JsonIgnore]
+    public float UsedIngestionCapacity
```

Comment:
Either the comment or the name of this property is incorrect. EngulfSize vs the old name for the amount of stuff ingested

Reply:
Fixed.